airflow/www/static/js/tree/Table.jsx

```diff
@@ -0,0 +1,168 @@
+  const upperCount = lowerCount + data.length - 1;
+
+  const {
+    getTableProps,
+    getTableBodyProps,
+    allColumns,
+    prepareRow,
+    page,
+    canPreviousPage,
+    canNextPage,
+    nextPage,
+    previousPage,
+    state: { pageIndex, sortBy },
+  } = useTable(
+    {
+      columns,
+      data,
+      pageCount,
+      manualPagination: !!manualPagination,
+      manualSortBy: !!setSortBy,
+      initialState: {
+        pageIndex: offset ? offset / pageSize : 0,
+        pageSize,
+      },
+    },
+    useSortBy,
+    usePagination,
+  );
+
+  const handleNext = () => {
+    nextPage();
+    if (setOffset) setOffset((pageIndex + 1) * pageSize);
+  };
+
+  const handlePrevious = () => {
+    previousPage();
+    if (setOffset) setOffset((pageIndex - 1 || 0) * pageSize);
+  };
+
+  useEffect(() => {
+    if (setSortBy) setSortBy(sortBy);
+  }, [sortBy, setSortBy]);
+
+  return (
+    <>
+      <ChakraTable {...getTableProps()}>
+        <Thead>
+          <Tr>
+            {allColumns.map((column) => (
+              <Th
+                {...column.getHeaderProps(column.getSortByToggleProps())}
+              >
+                {column.render('Header')}
+                {column.isSorted && (
+                  column.isSortedDesc ? (
+                    <TiArrowSortedDown aria-label="sorted descending" style={{ display: 'inline' }} size="1em" />
+                  ) : (
+                    <TiArrowSortedUp aria-label="sorted ascending" style={{ display: 'inline' }} size="1em" />
+                  )
+                )}
+                {(!column.isSorted && column.canSort) && (<TiArrowUnsorted aria-label="unsorted" style={{ display: 'inline' }} size="1em" />)}
+              </Th>
+            ))}
+          </Tr>
+        </Thead>
+        <Tbody {...getTableBodyProps()}>
+          {!data.length && !isLoading && (
+          <Tr>
+            <Td colSpan={2}>No Data found.</Td>
+          </Tr>
+          )}
+          {page.map((row) => {
+            prepareRow(row);
+            return (
+              <Tr
+                {...row.getRowProps()}
+                _odd={{ backgroundColor: oddColor }}
+                _hover={{ backgroundColor: hoverColor }}
+              >
+                {row.cells.map((cell) => (
+                  <Td
+                    {...cell.getCellProps()}
+                    py={3}
+                  >
+                    {cell.render('Cell')}
+                  </Td>
+                ))}
+              </Tr>
+            );
+          })}
+        </Tbody>
+      </ChakraTable>
+      <Flex alignItems="center" justifyContent="flex-start" my={4}>
+        <IconButton variant="ghost" onClick={handlePrevious} disabled={!canPreviousPage} aria-label="Previous Page">
+          <MdKeyboardArrowLeft />
+        </IconButton>
+        <IconButton variant="ghost" onClick={handleNext} disabled={!canNextPage} aria-label="Next Page">
+          <MdKeyboardArrowRight />
+        </IconButton>
```

Comment:
Apparently the icons an be passed as children, but the component API calls for the `icon` prop:

```suggestion
        <IconButton icon={<MdKeyboardArrowLeft />} variant="ghost" onClick={handlePrevious} disabled={!canPreviousPage} aria-label="Previous Page" />
        <IconButton icon={<MdKeyboardArrowRight />} variant="ghost" onClick={handleNext} disabled={!canNextPage} aria-label="Next Page" />
```